app/src/main/AndroidManifest.xml

```diff
@@ -1,25 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.wire.android">
+        package="com.wire.android">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <application
-        android:name=".WireApplication"
-        android:allowBackup="false"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
-        android:theme="@style/AppTheme.Light">
+            android:name=".WireApplication"
+            android:allowBackup="false"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsRtl="true"
+            android:theme="@style/AppTheme.Light">
 
         <activity
-            android:name=".LauncherActivity"
-            android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.Dark"
-            android:exported="true">
+                android:name=".ui.MainActivity"
```

Comment:
I think we should separate `LauncherActivity` and `MainActivity` as the second one should probably have all the content available to the already logged in user and some actions for instance related to registration, login or custom backend links should be handled before that 🤔

Reply:
We can set this up later, it's not crucial right now